Modules/SubModule_Import/xASL_imp_DCM2NII_Subject_CopyTempDir.m

```diff
@@ -11,20 +11,20 @@ function xASL_imp_DCM2NII_Subject_CopyAnalysisDir(nii_files, bClone2Source)
 %   n/a
 %                         
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% DESCRIPTION: Make a copy of analysisdir in sourcedir.
+% DESCRIPTION: Make a copy of temp dir in source dir.
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % EXAMPLE:     n/a
 %
 % __________________________________
 % Copyright 2015-2021 ExploreASL
 
-    %% Make a copy of analysisdir in sourcedir.
+    %% Make a copy of temp dir in source dir
     if bClone2Source
         if ~isempty(nii_files)
             for iFile=1:length(nii_files)
-                % replace 'analysis' by 'source'
-                [iStart, iEnd] = regexp(nii_files{iFile}, 'analysis');
+                % replace 'temp' by 'source'
```

Comment:
also here, do you mean temp gets replaced by rawdata? Source should stay source right?

Reply:
I just checked it out again. I think we actually never run this part right now :D 
Have to ask @jan-petr what we used it for. Right now it would copy the niftis and jsons to a `source` directory, like described in the header.